semanticdb/scalac/library/src/main/scala/scala/meta/internal/semanticdb/scalac/TextDocumentOps.scala

```diff
@@ -635,6 +635,21 @@ trait TextDocumentOps { self: SemanticdbOps =>
               case gtree: g.TypeTreeWithDeferredRefCheck =>
                 traverse(gtree.check())
 
+              case gtree: g.Literal
+                  if gtree.tpe != null && gtree.tpe.typeSymbol.name.decode
+                    .contentEquals("Class") && gtree.pos.isRange =>
+                val coLen = "classOf".length
+
+                val mpos = gtree.pos.toMeta
+                val mposFix = new m.Position.Range(mpos.input, mpos.start, mpos.start + coLen)
+
+                occurrences(mposFix) = "scala/Predef.classOf()."
+                binders += mposFix
+
+                val cpos =
+                  new m.Position.Range(mpos.input, mpos.start + coLen + 1, mpos.end - 1)
+                occurrences(cpos) = gtree.value.typeValue.typeSymbol.toSemantic
+                binders += cpos
```

Comment:
Is this line needed? `classOf` is not a definition

Reply:
same for `mposFix`

Reply:
Fixed.